core/templates/dev/head/components/RuleTypeSelectorDirective.js

```diff
@@ -82,7 +82,12 @@ oppia.directive('ruleTypeSelector', [function() {
         $(select2Node).select2('val', $scope.localValue);
 
         $(select2Node).on('change', function(e) {
+          $scope.localValue = e.val;
```

Comment:
I Agree, it was not in my initial fix but after checking the commit history I saw it was there and I thought I might be missing something so I added it.